### PR TITLE
feat: theme-added

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added dark mode.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code is clean
good to merge

**Review by sanjitsarkar — APPROVED** [review]
Code look clean and well structured.
You can merge

File: src/App.css [hunk 1]
```diff
@@ -28,6 +28,12 @@
   color: #61dafb;
 }
 
+.dark-theme {
+  background-color: #212121;
```
Comment (zoyatanweer) [inline src/App.css:32]: use var

**Review by zoyatanweer — APPROVED** [review]
code looks clean. good to merge

File: src/App.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/App.css:32]: Okay

**Pull request merged**